pkg/cmd/alias/set/set.go

```diff
@@ -43,21 +43,28 @@ func NewCmdSet(f *cmdutil.Factory, runF func(*SetOptions) error) *cobra.Command
 			for commands with mixed quotes or multiple lines.
 
 			If '--shell' is specified, the alias will be run through a shell interpreter (sh). This allows you
-			to compose commands with "|" or redirect with ">". Note that extra arguments following the alias
+			to compose commands with '|' or redirect with '>'. Note that extra arguments following the alias
 			will not be automatically passed to the expanded expression. To have a shell alias receive
-			arguments, you must explicitly accept them using "$1", "$2", etc., or "$@" to accept all of them.
-
-			Platform note: on Windows, shell aliases are executed via "sh" as installed by Git For Windows. If
-			you have installed git on Windows in some other way, shell aliases may not work for you.
+			arguments, you must explicitly accept them using '$1', '$2', etc., or '$@' to accept all of them.
 
 			Quotes must always be used when defining a command as in the examples unless you pass '-'
 			as the expansion parameter and pipe your command to 'gh alias set'.
+
+			**Platform notes for Windows:**
+
+			- Shell aliases are executed via 'sh' as installed by Git For Windows.
+			  If you have installed git in some other way, shell aliases may not work for you.
+			- If you are using a shell that is *not* Unix-like (e.g. Command Prompt, cmd.exe),
```

Comment:
I think we are literally just adding this note for cmd.exe users. I cannot think of a single other real-world use-case where someone would need to use double quotes instead of single quotes. So instead of saying something confusing like `non-Unix-like shell`, let's just say something like “Note for cmd.exe users: use double quotes instead of single quotes.” Keep it short and to the point.

Reply:
Ok, I can revise.